### ISSUE TITLE
3.0.0 release ?

### ISSUE THREAD

**Issue by sunderme** [texstudio-org/texstudio#1177]
### Environment
- TeXstudio: 3.0.0
- Qt: N/A
- OS: all
- TeX distribution: all

Any show stoppers for a 3.0.0 release ?

**Comment by benibela** [comment]
I got a mail that the terminal widget has crashed (SIGFPE). Perhaps it is better to only create the terminal when the tab is opened

**Comment by MeanSquaredError** [comment]
Latest github snapshot works reliably for me on Fedora 32 and Windows 7.

**Comment by axkibe** [comment]
Been using HEAD in production for while now.

**Comment by JackHack96** [comment]
Using latest beta3 without issues on Ubuntu 20.04

**Comment by blueice760** [comment]
Hi I updated to beta3 on Win7 and compile times are unusually long. I typed up the Hello World type document with 4 lines loading no packages and my compile time was between 5 to 7 seconds. Loading amsmath made the compile times even longer.

**Comment by sunderme** [comment]
check compilation on command line. compile,time is usually not determined by txs.

**Comment by blueice760** [comment]
Ok so I looked up how to time execution times for programs and found this [website ](https://stackoverflow.com/questions/673523/how-do-i-measure-execution-time-of-a-command-on-the-windows-command-line) which states that timing can be done using the: Measure-Command {somethinghere} on windows powershell. 

![screenshot](https://user-images.githubusercontent.com/69060222/89726328-99f18000-da19-11ea-9162-5edd2cabba8b.png)

The runtime is about 0.32s using the command line yet the execution time from texstudio beta3 is so slow. I got up to 5 mississippi's counting in my head waiting for the compilation to finish. I don't know it'll be useful but here is the log file too.

[test.log](https://github.com/texstudio-org/texstudio/files/5046818/test.log)
